misc/services/mobile-webui/mobile-webui-frontend/src/reducers/wfProcesses_status/picking.js

```diff
@@ -12,21 +12,119 @@ export const pickingReducer = ({ draftState, action }) => {
       return reduceOnUpdateQtyPicked(draftState, action.payload);
     }
 
+    case types.UPDATE_ALT_PICKING_STEP_QTY: {
```

Comment:
imho you could merge these 2 actions in one.

Reply:
PS: pls check `computeStepStatus`  function. It shall consider alternatives too.

Reply:
PS: computeStepStatus shall consider alt steps.